pkg/i18n/translations/pl.json

```diff
@@ -727,7 +727,7 @@
   "NoItemSelected": "Nie wybrano elementu",
   "SelectedItemIsNotABranch": "Wybrany element nie jest branch'em",
   "SelectedItemDoesNotHaveFiles": "Wybrany element nie ma plików do wyświetlenia",
-  "RangeSelectNotSupportedForSubmodules": "Zaznaczanie zakresu nie jest wspierane dla submodułów",
+  "MultiSelectNotSupportedForSubmodules": "Zaznaczanie zakresu nie jest wspierane dla submodułów",
```

Comment:
Please drop these changes (this one and zh-CN.json), we don't touch these files in PRs. They are updated through Crowdin.